store/instance.go

```diff
@@ -165,7 +165,12 @@ func (s *Store) CountInstanceGroupByEngineAndEnvironmentID(ctx context.Context,
 	rows, err := tx.PTx.QueryContext(ctx, `
 		SELECT engine, environment_id, COUNT(*)
 		FROM instance
-		WHERE row_status = $1
+		WHERE row_status = $1 AND (
+			CASE WHEN id <= 102
```

Comment:
nit: some boolean algebra here might be easier to understand.

(id <= 102 AND updater_id != 1) OR id > 102